src/scripts/interaction.js

```diff
@@ -1552,6 +1561,29 @@ function Interaction(parameters, player, previousState) {
           self.trigger(event);
         });
 
+        instance.on('click', function (event) {
+            const nonceEl = window.parent.document.getElementById('wpnonce');
+            $.ajax({
+                type: 'POST',
+                dataType: 'json',
+                url: '/wp-json/tapybl-reports/v1/interaction',
+                beforeSend: function(xhr) {
+                    xhr.setRequestHeader('X-WP-Nonce', nonceEl.value);
+                },
+                data: {
+                    VideoName: player.contentData.metadata.title,
```

Comment:
We should have consistent naming - either all lower case, camelcase, or _ separator - but not mixed.

If we use https://jsonapi.org/recommendations/#naming it should be camelCase

`data : {
   videoName: ...
   h5pId: ..
   interactionName: 
}`